src/datasource.ts

```diff
@@ -816,7 +816,7 @@ export default class IrondbDatasource extends DataSourceApi<IrondbQueryInterface
                     } catch (error) {
                         log(() => 'throwerr() failed to parse json from error.data.message. error: ' + error);
                     }
-                    if (message.user_error) {
+                    if (message && message.user_error) {
                         throw message.user_error.message + ', in query: ' + message.arguments.q;
                     } else if (error.statusText === 'Not Found') {
```

Comment:
These fallbacks to error.statusText only happen inside of the if(error.data.message) block.   This looks wrong to me.   Shouldnt' those fallbacks back outside so they can be used if there is no error.data.message?

Reply:
Commit below should address these concerns.